tests/Unit/Domain/Test_InterfaceItems.cpp

```diff
@@ -404,3 +405,39 @@ SPECTRE_TEST_CASE("Unit.Domain.InterfaceItems.PartialSlice", "[Unit][Domain]") {
             {Direction<dim>::lower_xi(), expected_xi},
             {Direction<dim>::upper_eta(), expected_eta}}));
 }
+
+namespace {
+struct SimpleBase : db::SimpleTag {
+  std::string name() noexcept { return "SimpleBase"; }
+  using type = int;
+};
+
+struct SimpleDerived : SimpleBase {
+  using base = SimpleBase;
+  std::string name() noexcept { return "SimpleDerived"; }
+  using type = int;
+};
+
+struct ComputeBase : db::BaseTag {};
+
+struct ComputeDerived : ComputeBase, db::ComputeTag {
+  using base = ComputeBase;
+  std::string name() noexcept { return "ComputeDerived"; }
+  static auto function(const int& arg) noexcept { return arg + 1.5; }
+  using argument_tags = tmpl::list<SimpleBase>;
```

Comment:
Could you check that having a compute item that has `ComputeBase` as its argument works correctly?

Reply:
Doesn't work, and I can't think of a way to fix it.  I may have to drop `db::BaseTag` support again, but I'll think about it for a few days first.